src/core_modules/capture-core/components/Pages/MainPage/EventsList/Pagination/ListPagination.container.js

```diff
@@ -0,0 +1,24 @@
+// @flow
+import { type ComponentType } from 'react';
+import { connect } from 'react-redux';
+import { changePage, changeRowsPerPage } from './pagination.actions';
+import { ListPaginationComponent } from './ListPagination.component';
+import type { DispatchersFromRedux, OwnProps, Props, PropsFromRedux } from './ListPagination.types';
+
+
+const mapStateToProps = (state: ReduxState, ownProps: OwnProps): PropsFromRedux => ({
+    rowsPerPage: state.workingListsMeta[ownProps.listId].rowsPerPage,
+    currentPage: state.workingListsMeta[ownProps.listId].currentPage,
+});
+
+const mapDispatchToProps = (dispatch: ReduxDispatch): DispatchersFromRedux => ({
+    onChangeRowsPerPage: (listId: string, rowsPerPage: number) => {
+        dispatch(changeRowsPerPage(listId, rowsPerPage));
+    },
+    onChangePage: (listId: string, page: number) => {
+        dispatch(changePage(listId, page));
+    },
+});
+
+export const ListPagination: ComponentType<OwnProps> =
+  connect<Props, OwnProps, _, _, _, _>(mapStateToProps, mapDispatchToProps)(ListPaginationComponent);
```

Comment:
replaced this `src/core_modules/capture-core/components/Pages/MainPage/EventsList/Pagination/withData.js`
with a container. Looked to me that this change moves this component closer to our style.